packages/experiments/src/components/Announced/Announced.ts

```diff
@@ -3,8 +3,4 @@ import { IAnnouncedProps, IAnnouncedStyleProps, IAnnouncedStyles } from './Annou
 import { AnnouncedBase } from './Announced.base';
 import { getStyles } from './Announced.styles';
 
-export const Announced = styled<IAnnouncedProps, IAnnouncedStyleProps, IAnnouncedStyles>(
-  AnnouncedBase,
-  getStyles,
-  undefined
-);
+export const Announced = styled<IAnnouncedProps, IAnnouncedStyleProps, IAnnouncedStyles>(AnnouncedBase, getStyles, undefined);
```

Comment:
I merged the master with the recent Prettier run over all packages, so I have no idea how this ended up being affected by my PR.